src/Settings.StyleCop

```diff
@@ -0,0 +1,65 @@
+<StyleCopSettings Version="105">
+  <GlobalSettings>
+    <CollectionProperty Name="RecognizedWords">
+      <Value>Chocolatey</Value>
+      <Value>em</Value>
+      <Value>Friedl's</Value>
+      <Value>hr</Value>
+      <Value>li</Value>
+      <Value>ol</Value>
+      <Value>runspace</Value>
+      <Value>sehe</Value>
+      <Value>stackoverflow</Value>
+      <Value>tunnelling</Value>
+      <Value>ul</Value>
+      <Value>xaml</Value>
+    </CollectionProperty>
+  </GlobalSettings>
+  <Analyzers>
+    <Analyzer AnalyzerId="StyleCop.CSharp.DocumentationRules">
+      <Rules>
+        <Rule Name="ElementsMustBeDocumented">
+          <RuleSettings>
+            <BooleanProperty Name="Enabled">False</BooleanProperty>
+          </RuleSettings>
+        </Rule>
+        <Rule Name="EnumerationItemsMustBeDocumented">
+          <RuleSettings>
+            <BooleanProperty Name="Enabled">False</BooleanProperty>
+          </RuleSettings>
+        </Rule>
+      </Rules>
+      <AnalyzerSettings>
+        <StringProperty Name="CompanyName">Chocolatey</StringProperty>
+        <StringProperty Name="Copyright">Copyright 2014 - Present Rob Reynolds, the maintainers of Chocolatey, and RealDimensions Software, LLC</StringProperty>
```

Comment:
> Ah, so keep the existing Copyright, and only apply the new copyright when files are added/edited?

Yes, that or we can get the same CLA in place for ChocolateyGUI and we can try to get all prior contributors to sign the CLA (all future will automatically get it). There are only 10 of you, so https://github.com/chocolatey/ChocolateyGUI/graphs/contributors